.github/workflows/deploy.yml

```
@@ -116,3 +90,45 @@ jobs:
             components install kubectl && gcloud composer environments run
             $COMPOSER_ENVIRONMENT --location $LOCATION variables import
             -- gcsfuse/actual_mount_path/variables.json
+
+  deploy-to-prod:
+    if: ${{ github.event.inputs.envName == 'prod' && github.ref == 'refs/heads/master' }}
+    runs-on: ubuntu-latest
+    needs: [tests]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install google-cloud-storage==2.1.0
+
+      - name: Authenticate to hubble GCP
+        uses: google-github-actions/auth@v1
+        with:
+          credentials_json: "${{ secrets.CREDS_PROD_HUBBLE }}"
+
+      - name: Upload files to prod GCS bucket
+        run: python dags/stellar_etl_airflow/add_files_to_composer.py --bucket $BUCKET --env prod
+        env:
+          GOOGLE_CLOUD_PROJECT: hubble-261722
+          BUCKET: us-central1-hubble-14c4ca64-bucket
+
+      - name: Update Airflow variables
+        uses: actions-hub/gcloud@master
+        env:
+          PROJECT_ID: hubble-261722
+          APPLICATION_CREDENTIALS: "${{ secrets.CREDS_TEST_HUBBLE }}"
```

Not sure if this was a mistake from the original prod .yml but I think this should be using `CREDS_PROD_HUBBLE` instead of `CREDS_TEST_HUBBLE`
```suggestion
          APPLICATION_CREDENTIALS: "${{ secrets.CREDS_PROD_HUBBLE }}"
```

---

Yeah it's an oopsie from the original ci-cd-prod.yml 😅

---

Oops, committed the suggestion